src/SourceGenerators/Uno.UI.Tasks/Content/Uno.UI.Tasks.targets

```diff
@@ -6,8 +6,30 @@
 		
 		<_IsXamlTrimmingAvailable>true</_IsXamlTrimmingAvailable>
 		<_IsXamlTrimmingAvailable Condition="'$(MSBuildRuntimeType)'!='Core' and !$([MSBuild]::IsOSPlatform('windows'))">false</_IsXamlTrimmingAvailable>
+		<_UnoLinkerHintGeneratorILLinkerPath>$(MSBuildThisFileDirectory)\..\tools\linker</_UnoLinkerHintGeneratorILLinkerPath>
 	</PropertyGroup>
 
+	<Choose>
+		<When Condition="
+			'$(TargetFrameworkIdentifier)'=='.NETCoreApp'
+			and $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '5.0'))
+			and '$(ILLinkTasksAssembly)'!=''">
+
```

Comment:
@jeromelaban I'm expecting this to always be true. So I think the condition could be removed (possible in a follow-up PR), and in this case the value `..\tools\linker` for path will be unused, and then we could actually just remove the linker we ship in the Uno package easily as it shouldn't exist there anyways :)

Reply:
It will be true but only net8.0 and later, and yes we'll be able to remove the linker we ship.

Reply:
Great. Can clean it up after https://github.com/unoplatform/uno/pull/16523 is merged